protobuf/model_config.proto

```diff
@@ -1734,6 +1734,37 @@ message ModelResponseCache
   bool enable = 1;
 }
 
+
```

Comment:
Should add the documentation on these fields as well.

Reply:
Yes, I will add them once I have finalize the structure